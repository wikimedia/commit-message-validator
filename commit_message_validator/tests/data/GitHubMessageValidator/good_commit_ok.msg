Replace thisFunction() with thatFunction()

This is a good commit subject.
