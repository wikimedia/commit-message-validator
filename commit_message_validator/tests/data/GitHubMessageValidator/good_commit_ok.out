commit-message-validator v%version%
Using GitHubMessageValidator to check the commit message
Commit message is formatted properly! Keep up the good work!
